Src/library/memory_analyzer/singletons/ma_singleton_factory.e

```diff
@@ -69,20 +69,6 @@ feature  -- Singletons
 			system_util_not_void: Result /= Void
 		end
 
-	main_window: MA_WINDOW
```

Comment:
See my previous comment, in MA_SHARED_MAIN_WINDOW.
And in addition, it was to avoid conflict with inherited `main_window`.